### PR TITLE
Rename maya-volume-exporter to maya-exporter

### PR DESCRIPTION
**Pull request by utkarshmani1997**
1. Why is this change necessary ?
- Refer to issue openebs/openebs#1379

2. How does this change address the issue ?
- Rename maya-volume-exporter to maya-exporter wherever used in the code.
- Rename the same in GNumakefile, buildscripts and Dockerfile.

3. How to verify this change ?

- Run `maya-exporter --help`
- You should get the output similar to given below
```
maya-exporter can be used to monitor openebs volumes and pools.
It can be deployed alongside the openebs volume or pool containers as sidecars.

Usage:
   [flags]

Examples:
maya-exporter -a=http://localhost:8001 -c=:9500 -m=/metrics

Flags:
      --alsologtostderr                  log to standard error as well as files
  -c, --controller.addr string           Address of the Jiva volume controller. (default "http://localhost:9501")
  -h, --help                             help for this command
  -a, --listen.addr string               Address on which to expose metrics and web interface.) (default ":9500")
  -m, --listen.path string               Path under which to expose metrics. (default "/metrics")
      --log-flush-frequency duration     Maximum number of seconds between log flushes (default 5s)
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files (default true)
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
```

4. What side effects does this change have ?
- none

5. Other details :
- Improvement: openebs/openebs#1379

Signed-off-by: Utkarsh Mani Tripathi <utkarshmani1997@gmail.com>

### CODE REVIEW

File: buildscripts/exporter/Dockerfile [hunk 1]
```diff
@@ -10,10 +10,10 @@ RUN apk add --no-cache \
 
 ARG BUILD_DATE
 LABEL org.label-schema.name="m-exporter"
-LABEL org.label-schema.description="OpenEBS Volume Exporter"
+LABEL org.label-schema.description="Maya Exporter"
```
Comment (kmova) [inline buildscripts/exporter/Dockerfile]: We could leave this as OpenEBS Maya Exporter

File: cmd/maya-exporter/app/command/exporter.go [hunk 2]
```diff
@@ -9,18 +9,19 @@ import (
...
 
-	"github.com/openebs/maya/cmd/maya-volume-exporter/app/collector"
+	"github.com/openebs/maya/cmd/maya-exporter/app/collector"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-// Entrypoint is used to monitor OpenEBS volumes. It's used by maya-agent to
-// start an instance of openebs volume exporter.
+// Entrypoint is used to monitor OpenEBS volumes. It's used to start an instance
```
Comment (kmova) [inline cmd/maya-exporter/app/command/exporter.go]: Can be changed to : Entrypoint is used to monitor OpenEBS volumes or pools.

File: cmd/maya-exporter/app/command/command.go [hunk 3]
```diff
@@ -46,10 +46,10 @@ func NewCmdVolumeExporter() (*cobra.Command, error) {
 	options.MetricsPath = "/metrics"
 	cmd := &cobra.Command{
 		Short: "Collect metrics from OpenEBS volumes",
-		Long: `  maya-volume-exporter monitors openebs volumes and exporter the metrics.
-  It starts collecting metrics from the jiva volume controller at the endpoint
-  "/v1/stats" Prometheus Server can collect the metrics from maya-volume-exporter.`,
-		Example: `  maya-volume-exporter -a=http://localhost:8001 -c=:9500 -m=/metrics`,
+		Long: `maya-exporter monitors openebs volumes and export the metrics.
+It starts collecting metrics from the jiva volume controller at the endpoint
```
Comment (kmova) [inline cmd/maya-exporter/app/command/command.go]: maya-exporter can be deployed alongside the openebs volume or pool containers as side cars.

File: cmd/maya-exporter/app/command/command.go [hunk 4]
```diff
@@ -46,10 +46,10 @@ func NewCmdVolumeExporter() (*cobra.Command, error) {
...
+"/v1/stats" Prometheus Server can collect the metrics from maya-exporter.`,
```
Comment (kmova) [inline cmd/maya-exporter/app/command/command.go]: remove the above line

**Comment by AmitKumarDas** [comment]
Review comments from @kmova are good. @utkarshmani1997 Please rework based on @kmova 's reviews.

**Comment by utkarshmani1997** [comment]
Thanks @AmitKumarDas @kmova for the review, i will push the changes soon.

**Pull request merged**